packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -0,0 +1,199 @@
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+/**
+ * Component to display dropdown list based on the options passed as parameters. The dropdown component is based
+ * on the Angular Material MatSelect.
+ */
+@Component({
+	selector: "stark-dropdown",
+	templateUrl: "./dropdown.component.html"
+})
+export class StarkDropdownComponent implements OnInit, OnChanges {
+
+	/**
+	 * (optional) if the dropdown will contain a default blank option
+	 */
+	@Input()
+	public defaultBlank?: boolean;
+	
+	/**
+	 * HTML "id" attribute of the element.
+	 */
+	@Input()
+	public dropdownId: string;
+	
+	/**
+	 * HTML "name" attribute of the element.
+	 */
+	@Input()
+	public dropdownName: string;
+	
+	/**
+	 * header Text to be displayed as the dropdown's header. Such header will be shown at the top of the
+	 * dropdown options list(dynamically translated via the $translate service if the provided text is defined in the translation keys).
+	 */
+	// TODO reenable once a solution for the replacement of md-select-header as been found
+	// @Input()
+	// public header?: string;
+	/**
+	 * (optional) Whether the dropdown is disabled
+	 */
+	@Input()
+	public isDisabled?: boolean;
+	
+	/**
+	 * Text to be displayed as the dropdown's label (dynamically translated via the $translate service if
+	 * the provided text is defined in the translation keys).
+	 */
+	@Input()
+	public label?: string;
+	
+	/**
+	 * (optional) Allows multiple option selection. Setting the attribute to "true" or empty
+	 * will enable this feature.
+	 */
+	@Input()
+	public multipleSelect?: string;
+	
+	/**
+	 * Array of options to be included in the dropdown list. This parameter is a one-way
+	 * binding (one-directional).
+	 */
+	@Input()
+	public options: any[];
+	
+	/**
+	 * (optional) Property name to be used as id of the options defined. If this parameter as
+	 * well as the optionLabelProperty are omitted, then the options parameter is considered to be an array of simple types.
+	 */
+	@Input()
+	public optionIdProperty?: string;
+	
+	/**
+	 * (optional) Property name to be used as label of the options defined (dynamically translated via
+	 * the $translate service if the provided text is defined in the translation keys). If this parameter as well as the
+	 * optionIdProperty are omitted, then the options parameter is considered to be an array of simple types.
+	 */
+	@Input()
+	public optionLabelProperty?: string;
+	
+	/**
+	 * Text to be displayed as the dropdown's placeholder (dynamically translated via the $translate
+	 * service if the provided text is defined in the translation keys).
+	 */
+	@Input()
+	public placeholder: string;
+	
+	/**
+	 * Source object to be bound to the dropdown ngModel.
+	 */
+	@Input()
+	public value: any | any[];
+	
+	/**
+	 * If the dropdown is required or not. by default, the dropdown is not required
+	 */
+	@Input()
+	public required: boolean = false;
+
+	/**
+	 * We will emit the newly selected value. 
```

Comment:
"We will emit" ? Could you please adapt this description and avoid the usage of "we" ? 😊